src/lib/store/remote/helpers.js

```diff
@@ -106,13 +108,21 @@ exports.parseFromRemote = function(state, object) {
   // as in some cases IDs might contain '/', too
   //
   matches = id.match(/([^\/]+)\/(.*)/);
+  if (matches == null) {
+    console.log('hoodie: unhandled id ' + id);
```

Comment:
I absolutely agree. Join the discussion here on the planned `hoodie.log` API https://github.com/hoodiehq/hoodie.js/issues/401